Cargo.toml

```diff
@@ -50,14 +50,14 @@ lazy_static = "1.4"
 bitflags = "1.2"
 smallvec = "1.0"
 parking_lot = "0.10"
-tokio = { version = "0.2", default-features=false, features=["rt-core", "rt-util", "io-driver", "tcp", "uds", "udp", "time", "signal", "sync"] }
+tokio = { version = "0.2", default-features = false, features=["rt-core", "rt-util", "io-driver", "tcp", "uds", "udp", "time", "signal", "sync"] }
 tokio-util   = { version = "0.2", features = ["full"] }
 
 # actix-http support
 actix-http = { version = "1.0", optional = true }
 
 # dns resolver
-trust-dns-resolver = { version = "0.18.0-alpha.2", optional = true, default-features = false }
+trust-dns-resolver = { version = "=0.18.0-alpha.2", optional = true, default-features = false }
```

Comment:
it would be very nice if you could write into a commit message what kind of dependency update you expect.

Reply:
I pushed an empty commit with a message explaining the situation. I hope this is what you were asking for.